src/bindings/python/tests_compatibility/test_ngraph/test_normalization.py

```diff
@@ -69,3 +69,18 @@ def test_mvn():
     assert node.get_output_size() == 1
     assert list(node.get_output_shape(0)) == [1, 3, 3, 3]
     assert node.get_output_element_type(0) == Type.f32
+
+
+def test_group_normalization():
+    data = ng.parameter((1, 3, 3, 3), name="data", dtype=np.float32)
+    scale = np.array([3], dtype=np.float32)
+    bias = np.array([3], dtype=np.float32)
+    num_groups = 2
+    epsilon = 1e-6
+
+    node = ng.group_normalization(data, scale, bias, num_groups, epsilon)
+
+    assert node.get_type_name() == "GroupNormalization"
+    assert node.get_output_size() == 1
+    assert list(node.get_output_shape(0)) == [1, 3, 3, 3]
+    assert node.get_output_element_type(0) == Type.f32
```

Comment:
This test is failing because incompatible shapes are used, also channel dimension should be divisible by num_groups.
https://dev.azure.com/openvinoci/dldt/_build/results?buildId=596884&view=logs&j=8e6fb299-7deb-52f1-2f4a-df91230aba00&t=f812e137-0344-51cb-e8b0-0ab752070263&l=4100